linkerd/app/outbound/src/lib.rs

```diff
@@ -402,17 +402,15 @@ impl Config {
         metrics: ProxyMetrics,
         span_sink: Option<mpsc::Sender<oc::Span>>,
         drain: drain::Watch,
-    ) -> impl tower::Service<
+    ) -> impl svc::NewService<
         listen::Addrs,
-        Error = impl Into<Error>,
-        Future = impl Send + 'static,
-        Response = impl tower::Service<
+        Service = impl tower::Service<
             I,
             Response = (),
             Error = impl Into<Error>,
             Future = impl Send + 'static,
         > + Send
-                       + 'static,
+                      + 'static,
```

Comment:
it is so weird that rustfmt decided to remove...one space...from these `'static` bounds???